README.md

```diff
@@ -67,11 +67,21 @@ python nlp/twitter.py -se
 python nlp/read.py
 ```
 
-## Process Tweets Pickle
+## Train Model Using Tweets Data
```

Comment:
Minor, but these all should be h3s, I accidentally made the Read/Process headers h2s, so you want to use ### instead of ##.